File: Binary_Tree_Inorder_Traversal.cpp
vector<int> inorderTraversal(TreeNode* root) {
    if(root == NULL){
        return {} ;
    }
    else{
        vector<int> result;
        for(int i: inorderTraversal(root->left)){
            result.push_back(i);
        }
        result.push_back(root->val);
        for(int i: inorderTraversal(root->right)){
            result.push_back(i);
        }
        return result;
    }
}

File: Binary_Tree_Inorder_Traversal.py
def inorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
    if root == None:
        return []
    else:
        a=[]
        for i in self.inorderTraversal(root.left):
            a.append(i)
        a.append(root.val)
    for i in self.inorderTraversal(root.right):
        a.append(i)
    return a
    
